slick/src/main/scala/slick/jdbc/H2Profile.scala

```diff
@@ -123,6 +127,32 @@ trait H2Profile extends JdbcProfile {
       override def valueToSQLLiteral(value: UUID) = "'" + value + "'"
       override def hasLiteralForm = true
     }
+    override val instantType : InstantJdbcType = new InstantJdbcType {
+      val formatter = new DateTimeFormatterBuilder()
```

Comment:
Can you add a comment why we need the formatter here. DB2 and Derby use `Instant.parse`, which can't this?

Reply:
Comment added